deploy/helm/sumologic/conf/instrumentation/otelcol.instrumentation.conf.yaml

```diff
@@ -205,4 +202,12 @@ service:
   telemetry:
     logs:
       level: {{ .Values.otelcolInstrumentation.logLevel }}
+    metrics:
```

Comment:
Added because of this deprecation - https://github.com/open-telemetry/opentelemetry-collector/pull/11205